di-ipv-queue-stub/enqueue-event/src/handlers/enqueueEvent.ts

```diff
@@ -70,16 +77,55 @@ export const handler: Handler = async (
                 ]
             }
             let addQueuePermissionCommand = new AddPermissionCommand(addQueuePermissionCommandInput);
-            let addQueuePermissionOutput:AddPermissionCommandOutput = await sqsClient.send(addQueuePermissionCommand);
+            let addQueuePermissionOutput: AddPermissionCommandOutput = await sqsClient.send(addQueuePermissionCommand);
```

Comment:
Is the console.log following this still required? Or should it give a better output explaining what it is?